src/features/metadata-based-view/__tests__/MetadataBasedItemList.test.js

```diff
@@ -169,6 +169,8 @@ describe('features/metadata-based-view/MetadataBasedItemList', () => {
             ${FIELD_TYPE_FLOAT}       | ${'123.456'}                  | ${123.456}
             ${FIELD_TYPE_MULTISELECT} | ${['Yes', 'No']}              | ${['Yes', 'No']}
             ${FIELD_TYPE_STRING}      | ${'str'}                      | ${'str'}
+            ${FIELD_TYPE_STRING}      | ${undefined}                  | ${undefined}
+            ${null}                   | ${'some value'}               | ${'some value'}
```

Comment:
Should we also test the cases in which the type is `FIELD_TYPE_INTEGER` or `FIELD_TYPE_FLOAT`, but the value is `null` or `undefined`?

Reply:
The additional two cases cover the function behavior where it returns `value` when either `value` or `type` doesn't exist. Checking for more types for the same result may not add additional testing.